src/parse-options.js

```diff
@@ -4,6 +4,8 @@ const DEFAULT_OPTIONS = {
   nodeBin: 'node',
   maxAttempts: 3,
   protractorArgs: [],
+  // set color to one of the colors available in 'chalk' - https://github.com/chalk/ansi-styles#colors
```

Comment:
We should handle converting this to a string or boolean value; we could also throw an error if the provided string is not a chalk supported color